packages/shell-api/api-extractor.json

```diff
@@ -15,7 +15,9 @@
     "@mongosh/arg-parser",
     "@mongosh/service-provider-core",
     "@mongosh/types",
-    "mongodb"
+    "mongodb",
+    "mongodb-schema",
+    "@types/json-schema"
```

Comment:
Not at all obvious, but this is actually a types-only package and json-schema is a "historical package" https://github.com/kriszyp/json-schema. Yet se import json-schema..